OpenSourcePolitics/decidim-cese
# fix translated field in edit initiative form

#### :tophat: Description
When editing an initiative after creating it, the description and the title appeared in the json storage way.
It now appears as a string in the good language.

As it now loads the initiative form in the initializer, and as the initiative form need active storage functions, we need to load active storage before.

The spec is the original one with the lines 27 and 28 added.

#### :pushpin: Related Issues
- [Notion card](https://www.notion.so/opensourcepolitics/bd4bf860eab94f9daca7da803f09e3b2?v=c96df65755dd403d95e86f82d7749709&p=de6e17ba5e9c4c5e9368a83c3ab38907&pm=c)

#### Testing
* Log in as user
* Create a petition 
* Edit your petition
* See the title and description field are displayed well

#### Tasks
